src/content/docs/style-guide/writing-docs/writer-workflow/github-intro.mdx

```diff
@@ -1,21 +1,23 @@
 ---
-title: Get around GitHub
+title: People, issues, and PRs
 topics:
   - Tech writer style guide
   - How to use GitHub
 redirects: 
   - /docs/style-guide/writer-workflow/github-intro
 ---
-As tech doc writers (TW) we [edit docs](/docs/style-guide/writer-workflow/tech-writer-workflow/), do [peer edits](/docs/style-guide/writer-workflow/peer-editor-workflow/), or use the [Docs Team GitHub board](https://github.com/newrelic/docs-website) to track the status of issues and pull requests (PR).  
+Tech writers (TW) are responsible for the docs. We [write and edit the docs](/docs/style-guide/writer-workflow/tech-writer-workflow/) and [exchange peer edits](/docs/style-guide/writer-workflow/peer-editor-workflow/), maintain our backlog, and `hero`. To make it all happen, we use our [Docs Team GitHub board](https://github.com/newrelic/docs-website) to track changes, file issues, and initiate pull requests (PRs).
 
-## Who is who in an issue/PR? [#who-is-who]
-GitHub keeps track of all activity concerning an issue or PR, including, of course, the people involved. When a new issue or PR is filed, check on the filer’s username and see if they're listed as a member of the New Relic organization. If they aren't, try to find them on Slack based on their username. If you're not sure about someone's affiliation, treat them as external until you know otherwise. 
+## The who's who of an issue or PR [#who-is-who]
+GitHub tracks every stage of an issue or pull request's life cycle, including the people behind the scenes.
 
-People in an issue/PR include:
+Because the Docs team invite anyone to file an issue or create a pull request through GitHub, we strongly recommend verifying a filer's relationship to the New Relic GitHub organization. If you're uncertain if a contributor is a Relic, a good trick is to check if they're a member of the New Relic GitHub org. Generally, we recommend assuming a filer is external until proven otherwise.
```

Comment:
I'm curious why you dropped the tip about looking on Slack

Reply:
there were a few reasons:

- at this juncture i wasn't sure who the audience of these docs were. i wasn't sure how a non-tw person who wanted to make edits would use slack, and it wasn't clear from what i had read, so...

- ...since we were 'getting around github,' focusing on using github as a one-stop shop tool for everything you'd need to know abt the entire process made the most sense. introducing a different tool (in this case, Slack) to check something felt out of scope. it also kinda added more information than needed if you could just check on GH.

thoughts? [thinky_face_emoji]

Reply:
I think it's good to keep in mind that the audience for this doc is pretty explicitly tech writers on the docs team. For this doc, I don't think we need to account for external readers at all. 

Slack is pretty much the easiest, quickest way to figure out if a contributor is at New Relic or not. It's what I use all the time to try to answer this question.